### PR TITLE
Implement acquireAADTokenService, setCanDrill in MockIVisualHost

### PR DESCRIPTION
**Pull request by adiletelf**
After updating to powerbi-visuals-api 5.7.0 `IVisualHost` has new fields `acquireAADTokenService`, `setCanDrill`.
PR is adding missing mocks and updating build.yml to use node 18, 20.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,13 +1,13 @@
-# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# This workflow will do a clean installation of node dependencies, build the source code and run tests across different versions of node
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
 
 name: build
 
 on:
   push:
-    branches: [main ]
+    branches: [ main, dev, certification ]
```
Comment (AleksSavelev) [inline .github/workflows/build.yml:8]: This is redundant. There is no dev and certification branches in utils

Comment (adiletelf) [reply .github/workflows/build.yml]: fixed

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -16,16 +16,21 @@ jobs:
...
       with:
         node-version: ${{ matrix.node-version }}
+    - run: npm audit
+      continue-on-error: true
     - run: npm ci
-    - run: npm run lint
+    - run: npm outdated
+      continue-on-error: true
+    - run: npm run lint --if-present
+    - run: npm run eslint --if-present
```
Comment (AleksSavelev) [inline .github/workflows/build.yml:33]: No need to add `npm run eslint`

**Review by AleksSavelev — CHANGES_REQUESTED** [review]

File: src/mocks/mockIAcquireAADTokenService.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import powerbi from "powerbi-visuals-api";
+import IAcquireAADTokenService = powerbi.extensibility.IAcquireAADTokenService;
+import AcquireAADTokenResult = powerbi.extensibility.AcquireAADTokenResult;
+import PrivilegeStatus = powerbi.PrivilegeStatus;
+
+export class MockIAcquireAADTokenService implements IAcquireAADTokenService {
+    acquireAADToken(): powerbi.IPromise<powerbi.extensibility.AcquireAADTokenResult> {
+        return new Promise<AcquireAADTokenResult>((resolve) => {
+            const token: AcquireAADTokenResult = { accessToken: undefined };
```
Comment (AleksSavelev) [inline src/mocks/mockIAcquireAADTokenService.ts:9]: Let's change the undefined to some test string

**Review by AleksSavelev — APPROVED** [review]

**Pull request merged**